tsconfig.json

```diff
@@ -1,3 +1,4 @@
 {
-  "extends": "astro/tsconfigs/strict"
+  "extends": "astro/tsconfigs/strict",
+  "compilerOptions": {"allowImportingTsExtensions": true},
```

Comment:
this shouldn't be needed if you call those files `parameters.tsx`, e.g.
However, you'll likely want to put something like
```json 
{
    "baseUrl": ".",
    "paths": {
      "@assets/*": ["src/assets/*"],
      "@lib/*": ["src/lib/*"],
      "@components/*": ["src/components/*"],
      "@layouts/*": ["src/layouts/*"],
      "@pages/*": ["src/pages/*"],
    },
}
```

into `compilerOptions`, to clean up all these imports (see above).